packages/katello/katello-installer/katello-installer-base.spec

```diff
@@ -2,16 +2,16 @@
 %undefine scl_prefix
 %global scl_ruby /usr/bin/ruby
 
-# %%global prever .rc1
+%global prerelease .rc1
 
 Name:    katello-installer-base
 Version: 3.8.0
-Release: 1.nightly%{?prever}%{?dist}
+Release: 0.1%{?prerelease}%{?dist}
 Summary: Puppet-based installer for the Katello and Katello Capsule
 Group:   Applications/System
 License: GPLv3+ and ASL 2.0
 URL:     https://theforeman.org/plugins/katello
-Source0: https://fedorapeople.org/groups/katello/releases/source/tarball/katello-installer-%{version}%{?prever}.tar.gz
+Source0: https://fedorapeople.org/groups/katello/releases/source/tarball/katello-installer-%{version}.tar.gz
```

Comment:
Shouldn't this still include the prerelease part? I'm also missing the annexed file here. If the build succeeds IMHO that indicates an issue by itself because our build pipeline should require it.